test/src/main/java/org/apache/accumulo/test/fate/FateOpsCommandsIT.java

```diff
@@ -0,0 +1,564 @@
+
+    // Start some transactions
+    FateId fateId1 = fate.startTransaction();
+    FateId fateId2 = fate.startTransaction();
+
+    ProcessInfo p = getCluster().exec(Admin.class, "fate", fateId1.canonical(), fateId2.canonical(),
+        "--summary", "-s", "NEW", "-t", store.type().name());
+    assertEquals(0, p.getProcess().waitFor());
+    String result = p.readStdOut();
+    assertTrue(result.contains("Status Filters: [NEW]"));
+    assertTrue(result
+        .contains("Fate ID Filters: [" + fateId1.canonical() + ", " + fateId2.canonical() + "]")
+        || result.contains(
+            "Fate ID Filters: [" + fateId2.canonical() + ", " + fateId1.canonical() + "]"));
+    assertTrue(result.contains("Instance Types Filters: [" + store.type().name() + "]"));
+  }
+
+  @Test
+  public void testFatePrintCommand() throws Exception {
+    executeTest(this::testFatePrintCommand);
+  }
+
+  protected void testFatePrintCommand(FateStore<TestEnv> store, ServerContext sctx)
+      throws Exception {
+    // Configure Fate
+    Fate<TestEnv> fate = initializeFate(store);
+    // Occasionally, the summary/print cmds will see a COMMIT_COMPACTION transaction which was
+    // initiated on starting the manager, causing the test to fail. Stopping the compactor fixes
+    // this issue.
+    getCluster().getClusterControl().stopAllServers(ServerType.COMPACTOR);
+    Wait.waitFor(() -> getCompactorAddrs(getCluster().getServerContext()).isEmpty(), 60_000);
+
+    // validate no transactions
+    ProcessInfo p = getCluster().exec(Admin.class, "fate", "--print");
+    assertEquals(0, p.getProcess().waitFor());
+    String result = p.readStdOut();
+    assertTrue(result.contains("0 transactions"));
+
+    // create Fate transactions
+    FateId fateId1 = fate.startTransaction();
+    FateId fateId2 = fate.startTransaction();
+
+    // Get all transactions. Should be 2 FateIds with a NEW status
+    p = getCluster().exec(Admin.class, "fate", "--print");
+    assertEquals(0, p.getProcess().waitFor());
+    result = p.readStdOut();
+    Map<String,String> fateIdsFromResult = getFateIdsFromPrint(result);
+    assertEquals(Map.of(fateId1.canonical(), "NEW", fateId2.canonical(), "NEW"), fateIdsFromResult);
+
+    /*
+     * Test filtering by States
+     */
+
+    // Filter by NEW state
+    p = getCluster().exec(Admin.class, "fate", "--print", "-s", "NEW");
+    assertEquals(0, p.getProcess().waitFor());
+    result = p.readStdOut();
+    fateIdsFromResult = getFateIdsFromPrint(result);
+    assertEquals(Map.of(fateId1.canonical(), "NEW", fateId2.canonical(), "NEW"), fateIdsFromResult);
+
+    // Filter by FAILED state
+    p = getCluster().exec(Admin.class, "fate", "--print", "-s", "FAILED");
+    assertEquals(0, p.getProcess().waitFor());
+    result = p.readStdOut();
+    fateIdsFromResult = getFateIdsFromPrint(result);
+    assertTrue(fateIdsFromResult.isEmpty());
+
+    /*
+     * Test filtering by FateIds
+     */
+
```

Comment:
Could test filtering by a fateId that does not exist in the store and verify nothing is returned.